pkg/middleware/unimplemented.go

```diff
@@ -21,7 +21,7 @@ func (b *UnimplementedMiddleware) Next(req *proto.Data) (*proto.Data, error) {
 	resp := req
```

Comment:
You don't need this variable anymore, you can just return req in the latest return